include/revng/Pipeline/Invokable.h

```diff
@@ -168,22 +168,26 @@ using ConcatImpl = std::decay_t<decltype(*concatImpl(std::declval<T1 *>(),
                                                      std::declval<T2 *>()))>;
 
 template<typename First, typename... Rest>
-auto *concantMultiImp(const First *, const Rest *...Others) {
+auto *multiConcatImpl(const First *, const Rest *...Others) {
   if constexpr (sizeof...(Rest) == 0) {
     return static_cast<First *>(nullptr);
   } else {
-    auto *Recurred = concantMultiImp<Rest...>(Others...);
+    auto *Recurred = multiConcatImpl<Rest...>(Others...);
     using TupleType = std::decay_t<decltype(*Recurred)>;
     using Concatted = ConcatImpl<First, TupleType>;
     return static_cast<Concatted *>(nullptr);
   }
 }
 
+inline std::tuple<> *multiConcatImpl() {
+  return nullptr;
+}
+
 template<typename T>
 using Decay = std::decay_t<T>;
 
 template<typename... T>
-using TupleConcat = Decay<decltype(*concantMultiImp(std::declval<T *>()...))>;
+using TupleConcat = Decay<decltype(*multiConcatImpl(std::declval<T *>()...))>;
 
```

Comment:
Can you make a more explicative commit message?

Reply:
What do you think?
https://github.com/revng/revng/pull/232/commits/667f6cefb8bfb4fce1abe336ad9c4a1e70f33536